mgrctl/cmd/api/login.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2024 SUSE LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/uyuni-project/uyuni-tools/shared/api"
+	. "github.com/uyuni-project/uyuni-tools/shared/l10n"
+	"github.com/uyuni-project/uyuni-tools/shared/types"
+	"github.com/uyuni-project/uyuni-tools/shared/utils"
+)
+
+func runLogin(globalFlags *types.GlobalFlags, flags *apiFlags, cmd *cobra.Command, args []string) error {
+	log.Debug().Msg("Running login command")
+
+	utils.AskIfMissing(&flags.User, cmd.Flag("api-user").Usage, 0, 0, nil)
+	utils.AskPasswordIfMissing(&flags.Password, cmd.Flag("api-password").Usage, 0, 0)
+	// ToDO add FQDN checker from rebase
```

Comment:
```suggestion
	// TODO add FQDN checker from rebase
```
Most editors highlight `TODO` comments all caps, not sure about mixed casing

Reply:
Ah, I forgot about this. Now with all PRs merged and want to use https://github.com/uyuni-project/uyuni-tools/blob/main/shared/utils/utils.go#L411